promotion/src/main/resources/application.yml

```
@@ -43,4 +43,16 @@ eureka:
+
+management:
+  endpoints:
+    web:
+      exposure:
+        include: prometheus, health, info
+  tracing:
+    sampling:
+      probability: 1.0
+  zipkin:
+    tracing:
+      endpoint: http://localhost:9411/api/v2/spans
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)